apps/meteor/app/livechat/server/lib/hooks.ts

```diff
@@ -119,3 +120,6 @@ export const afterTakeInquiry = makeFunction(
 		await sendToCRM('LivechatSessionTaken', room);
 	},
 );
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export const afterInquiryQueued = makeFunction(async (_inquiry: ILivechatInquiryRecord) => {});
```

Comment:
you gotta be smart
```suggestion
export const afterInquiryQueued = makeFunction(async (_inquiry: ILivechatInquiryRecord) => {
	// no op
});
```

this or either `return null` or even `return void 0` why not? I think any of these are better than messing up with eslint rules